src/utils/misc-utils.ts

```diff
@@ -61,7 +61,7 @@ export const splitPathAndName = (stringVal: string): string[] => {
     if (!stringVal) return [];
 
     const lastSlash = stringVal.lastIndexOf('/') + 1;
-    return [stringVal.substr(0, lastSlash), stringVal.substr(lastSlash)];
+    return [stringVal.substring(0, lastSlash), stringVal.substring(lastSlash)];
```

Comment:
This is the right call - but making sure you're aware these are different calls now. I don't think they have any consequence for us though.